src/main/java/frc/robot/subsystems/Drive.java

```diff
@@ -146,6 +147,77 @@ public void end(boolean interrupted) {
         };
     }
 
+    public CommandBase setAbsoluteAngle(final Rotation2d angle, final double speed) {
```

Comment:
All comments on `setRelativeAngle` also apply here.

Are the functions similar enough that you can extract common logic?